<launch>

<node pkg="rqt_image_view" type="rqt_image_view" name="image_view_cropped" args="/image_cropped" />
<node pkg="rqt_image_view" type="rqt_image_view" name="image_view_white" args="/image_white" />
<node pkg="rqt_image_view" type="rqt_image_view" name="image_view_yellow" args="/image_yellow" />
<node pkg="image_crop" type="image_crop_hw7.py" name="image_cropper" />
<node pkg="image_processing_hw" type="image_pub_all.py" name="image_pub_all" />

</launch>
